include/arkode/arkode_arkstep.h

```diff
@@ -159,6 +161,7 @@ SUNDIALS_EXPORT int ARKStepRootInit(void *arkode_mem, int nrtfn,
 
 /* Optional input functions -- must be called AFTER ARKStepCreate */
 SUNDIALS_EXPORT int ARKStepSetDefaults(void* arkode_mem);
+SUNDIALS_DEPRECATED_EXPORT_MSG("use individual controls instead")
 SUNDIALS_EXPORT int ARKStepSetOptimalParams(void *arkode_mem);
```

Comment:
Here and elsewhere `SUNDIALS_DEPRECATED_EXPORT_MSG` should replace `SUNDIALS_EXPORT`

Reply:
Does this function need to be deprecated with the change to controller objects or is being deprecated on its own?

Reply:
It is being deprecated for three reasons: 
1. The new controller and heuristics objects are the natural places to modify these values.
2. It is now clear that the "defaults" my optimization method honed in on years ago were biased toward very stiff applications with direct algebraic solvers, and thus the name implies some meaningless guarantee.
3. I've never heard of a single user calling this function.